test_translate_modules doesn't work

Please could you take a look at https://drone.gisce.net/gisce/erp/5020

Especially to:
```
2017-07-03 09:15:29,584:INFO:destral.utils.compare_pofiles:Could not get po/pot file: /drone/src/github.com/gisce/erp/server/bin/addons/giscedata_perfils/i18n/[.po

AssertionError: There are -1 missing strings in the PO file of the module giscedata_perfils
```

But maybe is a testing suite configuration problem.

--- a/destral/testing.py
+++ b/destral/testing.py
@@ -215,13 +215,16 @@ class OOBaseTests(OOTestCase):
                     missing_strings, self.config['module']
                 )
             )
+            logger.info('Checking translations for langs: {}'.format(
+                self.config['testing_langs']
+            ))
             for test_lang in self.config['testing_langs']:
                 po_path = join(trad_path, '{}.po'.format(test_lang))
                 missing_strings, untranslated_strings = compare_pofiles(
                     tmp_pot, po_path
                 )
                 self.assertFalse(
-                    untranslated_strings,
+                    missing_strings,
                     'There are {} missing strings in the PO file'
                     ' of the module {}'.format(
                         untranslated_strings, self.config['module']

--- a/destral/utils.py
+++ b/destral/utils.py
@@ -212,14 +212,6 @@ def compare_pofiles(pathA, pathB):
             continue
         if not msgB.string:
             not_translated += 1
-    if not_found:
-        logger.info("There are {} strings from {} missing in {}".format(
-            not_found, pathA, pathB
-        ))
-    if not_translated:
-        logger.info("There are {} strings missing translation in {}".format(
-            not_translated, pathB
-        ))
     return not_found, not_translated
 
 

